cherryontech/JourneyWise
# refactored the archetype

## Pull Request

### Description
Refactored the ArcheType component so it can be more dynamic 

### Related Issue
worked on the issue https://marisolsaumi.atlassian.net/browse/JW-50

### Description of Changes
Made the the image and text to a different component. 
Made the dots into a different component 
Made the userCard to different component
Add the correct data for the json file 

### Checklist
- [x ] I have read the CONTRIBUTING.md document.
- [ x] My code follows the code style of this project.
- [x ] My changes generate no new warnings or errors.
- [x ] I have updated the documentation or comments where necessary.
- [ x] All tests pass locally.

### Testing Instructions
Clone the branch to your local repository and run it to see if all is working properly. 

### Screenshots (if applicable)
<img width="760" alt="Screenshot 2024-05-16 at 03 04 39" src="https://github.com/cherryontech/JourneyWise/assets/77724559/4bdad685-89c
<img width="510" alt="Screenshot 2024-05-16 at 03 04 47" src="https://github.com/cherryontech/JourneyWise/assets/77724559/ce63419a-07bf-495b-84ac-7d6d2d6187f7">
0-4268-8b54-d37e0857fb98">
<img width="510" alt="Screenshot 2024-05-16 at 03 04 47" src="https://github.com/cherryontech/JourneyWise/assets/77724559/2114906e-e3bc-4b42-ae50-da3216fb21be">


### Gotchas/Learnings
Still getting used to tailwind 
### Additional Notes (Optional)
* Include any other relevant information for reviewers.
